### ISSUE TITLE
folder based testing breaks on iosxe -> ShowIpBgpNeighborsAdvertisedRoutes -> golden_output4

### ISSUE THREAD

**Issue by vkoulermos** [CiscoTestAutomation/genieparser#621]
When running tests using `python folder_parsing_job.py -o iosxe` I get this:

```
Traceback (most recent call last):
   File "/pyatsdev/genieparser/src/genie/libs/parser/utils/unittests.py", line 574, in test_golden
     parsed_output = obj.parse(**arguments)
   File "src/genie/metaparser/_metaparser.py", line 299, in genie.metaparser._metaparser.MetaParser.parse
   File "src/genie/metaparser/_metaparser.py", line 292, in genie.metaparser._metaparser.MetaParser.parse
   File "src/genie/metaparser/util/schemaengine.py", line 229, in genie.metaparser.util.schemaengine.Schema.validate
 genie.metaparser.util.exceptions.SchemaEmptyParserError: Parser Output is empty
```

I am working on some iosxe show_bgp.py parsers and to bypass this issue I removed the all the golden_output4* files from this directory `genieparser/src/genie/libs/parser/iosxe/tests/ShowIpBgpNeighborsAdvertisedRoutes/cli/equal` and all other tests seem to work properly

**Comment by vkoulermos** [comment]
I tested and the bug is present even without the any changes. I checked the latest change on iosxe show_bgp.py PR #607 . Same error:
<img width="1429" alt="Screenshot 2022-02-25 at 6 03 17 PM" src="https://user-images.githubusercontent.com/99613649/155748738-6e2f00fd-e53d-48c1-9cf6-a5f07043b042.png">

**Comment by ThomasJRyan** [comment]
I am unable to replicate this on the latest `master` branch. This was either resolved in the latest release or it is specific to your branch only

**Comment by vkoulermos** [comment]
It has been fixed in #625. Same unittests removed as well. Thanks!
